pom.xml

```diff
@@ -303,7 +303,6 @@
                 <groupId>ch.qos.logback</groupId>
                 <artifactId>logback-classic</artifactId>
                 <version>${logback.version}</version>
-                <scope>test</scope>
```

Comment:
This impacts all other modules. Now ogm-core would ship with logback dependency which is probably not intended.
Maybe instead explicitly set scope to compile in neo4j-ogm-tests/neo4j-ogm-integration-tests/pom.xml?

Reply:
Very good catch, if it would be in `<dependencies />`, but it is in `<dependencyManagement />`. I needed to relax the scope, so that I could move it into compile scope in the `neo4j-ogm-integration-tests` module. We have a bunch of utility classes in it's `main` folder already, so having the capturing log rule there is consistent. We have stopped deploying the test module with 3.2, it is not part of public api.

Reply:
I was talking about other modules (e.g. ogm-core) where test scope is not set, so scope is inherited from parent's dependency management.
Maybe I misread something?